src/Common/Auth/AuthUtils.php

```diff
@@ -262,9 +263,10 @@ private function confirmPatientPassword($username, &$password, $email = '')
 
     /**
      *
-     * @param type $username
-     * @param type $password - password is passed by reference so that it can be "cleared out" as soon as we are done with it.
+     * @param $username
+     * @param $password - password is passed by reference so that it can be "cleared out" as soon as we are done with it.
      * @return boolean  returns true if the password for the given user is correct, false otherwise.
+     * @throws SodiumException
      */
     private function confirmUserPassword($username, &$password)
```

Comment:
Maybe I'll hook here somewhere!

Reply:
postmaster update testing well so I'll move on to notifying admin and user if nearing password expiring.
Of note: PHPMailer requires valid certificates and path set in php.ini setting
 `curl.cainfo="C:\xampp\apache\bin\curl-ca-bundle.crt"` or correct path to certificate.